app/views/articles/show.html.slim

```diff
@@ -0,0 +1,17 @@
+h1
+  | News №#{@article.id}
+
+= image_tag(@article.avatar_url(:thumb)) if @article.avatar.present?
```

Comment:
I propose you to move this line to helper. Helper is a method you can use in a view.

Let me know if you need any help to create a helper method

Reply:
Added show_image method to application_helper.rb